charts/dapr/charts/dapr_operator/templates/dapr_operator_deployment.yaml

```diff
@@ -83,11 +83,11 @@ spec:
         - "--log-as-json"
 {{- end }}
 {{- if eq .Values.global.prometheus.enabled true }}
-        - "--enable-metrics"
+        - "--enable-metrics-exporter"
```

Comment:
Why did we add -exporter? I think just --enable-metrics is more clear.

Reply:
@yaron2 , enable-metrics flag is actually starting the exporter and not actually enabling/init the metrics. Till now, metrics is default turned on, metrics is getting enabled under the same flag. So I discussed with @youngbupark  to make a change in the name as well.  I added "-exporter" for clarity but I am open to keep that way also.  @youngbupark , your thoughts ?

Reply:
ok, this is for the operator settings right? I'm fine to keep it as is.

Reply:
Yes, this one you pointed out is for operator settings.